docs/api-reference/gltf-loaders/gltf-loader.md

```diff
@@ -2,22 +2,52 @@
 
 Parses a glTF file into a hirearchical scenegraph description that can be used to instantiate an actual Scenegraph in most WebGL libraries.
 
-Can load a binary GLB chunk and decodes it into a JavaScript data structure and a blob with binary data.
+| Loader                     | Characteristic |
+| ---                        | ---            |
+| File Extension             | `.glb`,`.gltf` |
+| File Type                  | Binary         |
+| File Format                | [glTF](https://github.com/KhronosGroup/glTF/tree/master/specification/2.0) |
+| Parser Category            | glTF Scenegraph |
+| Parser Type                | Asynchronous (Synchronous w/ limited functionality) |
+| Worker Thread Support      | Not yet        |
+| Streaming Support          | No             |
 
 
 ## Usage
 
 ```
-import {GLTFLoader} from '@loaders.gl/gltf';
 import {loadFile} from '@loaders.gl/core';
-
-loadFile(url, GLTFLoader).then(data => {
-  // Application code here
-  ...
-});
+import {GLTFLoader} from '@loaders.gl/gltf';
+const gltf = await loadFile(url, GLTFLoader);
 ```
 
+## Options
+
+* `DracoEncoder` - supply this to enable decoding of Draco compressed meshes.
+
 
 ## Structure of Loaded Data
 
 Returns a JSON object with "embedded" binary data in the form of typed javascript arrays.
+
+
+## Attributions
+
+The `GLTFLoader` was written specifically for loaders.gl.
+
+
+## Additional Information
+
+Can load both binary `.glb` files and JSON `.gltf` files.
+
+When parsed asynchronously (not using `loadSync` or `parseSync`):
+* linked binary resources will be loaded and resolved (if url is available).
+* base64 encoded binary data inside the JSON payload will be decodec
+
+To support decoding of Draco compressed meshes:
+```
+import {loadFile} from '@loaders.gl/core';
+import {GLTFLoader} from '@loaders.gl/gltf';
+import {DracoDecoder} from '@loaders.gl/draco';
+const gltf = loadFile(url, GLTFLoader, {DracoDecoder});
```

Comment:
Move to gltf-parser.md?

Reply:
> Move to gltf-parser.md?

Will copy the same information in my coming Draco-focused diff.

The primary API we export is the `GLTFLoader`, the `GLTFParser` is more a helper class.